src/AppBundle/EventListener/RedirectToPreferredLocaleSubscriber.php

```diff
@@ -23,7 +25,7 @@
  *
  * @author Oleg Voronkovich <oleg-voronkovich@yandex.ru>
  */
-class RedirectToPreferredLocaleListener
+class RedirectToPreferredLocaleSubscriber implements EventSubscriberInterface
```

Comment:
We may encourage to produce `EventSubscriber`s in favor of regular `EventListener`s. We can indeed simplify the `resource` to regiter all classes in the `EventListener/` directory. Because later we also override the listener class to complete its definition.